pipeline/alert.go

```diff
@@ -1254,3 +1258,76 @@ func (a *AlertNode) Talk() *TalkHandler {
 type TalkHandler struct {
 	*AlertNode
 }
+
+// Send the alert using SNMP traps.
+// To allow Kapacitor to post SNMP traps,
+//
+// Example:
+//    [snmp]
+//      enabled = true
+//      target-ip = "127.0.0.1"
+//      target-port = 9162
+//      community = "public"
+//      version = "2c"
+//      state-changes-only = false
+//
+// In order to not post a message every alert interval
+// use AlertNode.StateChangesOnly so that only events
+// where the alert changed state are posted to the channel.
+//
+// Example:
+//    stream
+//         |alert()
+//             .snmp()
+//             .trap('1.3.6.1.2.1.1.7', 'i', {{ Index field value }})
+//
+// Send alerts to `target-ip:target-port` on OID '1.3.6.1.2.1.1.7'
+//
+// tick:property
+func (a *AlertNode) Snmp(trap ...[]interface{}) *SnmpHandler {
+	snmp := &SnmpHandler{
+		AlertNode: a,
+		TrapList:  trap,
+	}
+	a.SnmpHandlers = append(a.SnmpHandlers, snmp)
+	return snmp
+}
+
+// Email AlertHandler
+// tick:embedded:AlertNode.Email
+type SnmpHandler struct {
+	*AlertNode
+
+	// List of email recipients.
+	// tick:ignore
+	TrapList [][]interface{} `tick:"Trap"`
+}
+
+// Define the To addresses for the email alert.
+// Multiple calls append to the existing list of addresses.
+// If empty uses the addresses from the configuration.
+//
+// Example:
+//    |alert()
+//       .id('{{ .Name }}')
+//       // Email subject
+//       .meassage('{{ .ID }}:{{ .Level }}')
+//       //Email body as HTML
+//       .details('''
+//<h1>{{ .ID }}</h1>
+//<b>{{ .Message }}</b>
+//Value: {{ index .Fields "value" }}
+//''')
+//       .email('admin@example.com')
+//         .to('oncall@example.com')
+//         .to('support@example.com')
+//
+// All three email addresses will receive the alert message.
+//
+// Passing addresses to the `email` property directly or using the `email.to` property is the same.
+// tick:property
+func (h *SnmpHandler) Trap(trap ...interface{}) *SnmpHandler {
```

Comment:
This comment needs to be updated to be SNMP related, its still the email comments.